### PR TITLE
Hide editor links if viewing the project as a platform admin

### LINKED ISSUE #1028: Remove open editor links if viewing a project as an administrator
### Description

If the user is viewing the project page because they are an admin and not a member of the team they have access to all functionality with the exception of opening the editor.

Currently if they do load the editor Node-RED sits in a retry loop without any obvious feedback (to fix upstream)

We should remove the links to Open Editor and not make the Editor URL clickable if the user is viewing as an admin to further re-enforce this limitation
image

### Epic/Story

_No response_

### PR DESCRIPTION
**Pull request by Pezmc**
Follow up to https://github.com/flowforge/flowforge/issues/940, fixes #1028 

### After

Screenshot 2022-10-11 at 16 11 22

### Before

Screenshot 2022-10-11 at 16 11 44

I went on a deep dive of the e2e tests with cypress, before ultimately concluding this doesn't need explicit coverage.

### CODE REVIEW

File: frontend/src/pages/project/index.vue [hunk 1]
```diff
@@ -22,7 +22,7 @@
             </template>
             <template v-slot:tools>
                 <div class="space-x-2 flex">
-                    <a v-if="editorAvailable" :href="project.url" target="_blank" class="ff-btn ff-btn--secondary" data-action="open-editor">
+                    <a v-if="editorAvailable && !isVisitingAdmin" :href="project.url" target="_blank" class="ff-btn ff-btn--secondary" data-action="open-editor">
```
Comment (Pezmc) [inline frontend/src/pages/project/index.vue:25]: I was originally included to disable the button (and add a title/tooltip explaining why), but decided against this after discussing with @joepavitt.

File: frontend/src/pages/project/Overview.vue [hunk 2]
```diff
@@ -8,7 +8,15 @@
                     <tr class="border-b">
                         <td class="w-1/4 font-medium">Editor</td>
                         <td>
-                            <a v-if="editorAvailable" :href="project.url" target="_blank" class="forge-button-secondary py-1 mb-1"><span class="ml-r">{{project.url}}</span><ExternalLinkIcon class="w-4 ml-3" /></a>
+                            <div v-if="editorAvailable">
+                                <div v-if="isVisitingAdmin" class="my-2">
```
Comment (joepavitt) [inline frontend/src/pages/project/Overview.vue:12]: This suggests in your "after" screenshot, we should be seeing the `project.url`, but I do not?

Comment (Pezmc) [reply frontend/src/pages/project/Overview.vue:12]: Unless I'm missing something, that's the `http://localhost:7881`

Comment (joepavitt) [reply frontend/src/pages/project/Overview.vue:12]: yep - sorry, you're absolutely right, and I just deleted my comment in realising that :D

**Review by joepavitt — APPROVED** [review]

**Pull request merged**